CHANGELOG.md

```diff
@@ -146,4 +153,4 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 [1.0.0]: https://github.com/cyberark/conjur-service-broker/compare/v0.3.1...v1.0.0
 [0.3.1]: https://github.com/cyberark/conjur-service-broker/compare/v0.3.0...v0.3.1
 [0.3.0]: https://github.com/cyberark/conjur-service-broker/compare/v0.2.0...v0.3.0
-[0.2.0]: https://github.com/cyberark/conjur-service-broker/compare/v0.1.0...v0.2.0
+[0.2.0]: https://github.com/cyberark/conjur-service-broker/compare/v0.1.0...v0.2.0
```

Comment:
Not sure why this like is changed, It must have a hidden character. Can you revert it?

Reply:
Change log looks good now @rpothier 

```
SC-MKEFFEL-OSX:conjur-service-broker mkeffele$ ./dev/parse-changelog.sh
++ dirname ./dev/parse-changelog.sh
+ cd ./dev
+ docker run --rm -v /Users/mkeffele/git_repos/conjur-service-broker/dev/..:/work -w /work ruby:2.5 bash -ec '
    gem install -N parse_a_changelog
    parse ./CHANGELOG.md
  '
Successfully installed polyglot-0.3.5
Successfully installed treetop-1.6.11
Successfully installed parse_a_changelog-1.1.0
3 gems installed
Parsing ./CHANGELOG.md
SUCCESS
```